docs/api.md

```diff
@@ -373,12 +424,45 @@ endpoint) if any of these are true:
     "region_code": "01",
     "scheme_member": "Y",
     "recruitment_cycle": "2019",
+    "g12_contact_address": "info@asmescitt.education.uk",
     "campuses": [
       {
         "campus_code": "",
         "name": "Main Site",
         "region_code": "01"
       }
+    ],
+    "contacts": [
```

Comment:
these ain't actual people and contact right?

Reply:
No they aren't, I mean, not that I'm aware. Guess I could double-check.

Reply:
I've made these obviously fake now.